.github/workflows/fedora.yml

```diff
@@ -24,6 +24,8 @@ jobs:
           dnf -y install copr-cli make gcc-c++
           dnf -y upgrade git
           dnf -y install perl
+          dnf -y install libffi-devel
+          dnf -y install zlib-devel
```

Comment:
maybe need install xft package on fedora. xmake-repo has not added this package yet

Reply:
or we can add libxft in xmake-repo. It seems it is not very complicated

https://github.com/Homebrew/linuxbrew-core/blob/master/Formula/libxft.rb